sample/test-sample/frontend/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@eclipse-sirius/sirius-web",
+  "author": "Eclipse Sirius",
+  "version": "0.1.0",
+  "license": "EPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-sirius/sirius-web"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com/"
+  },
+  "private": true,
```

Comment:
Same remark: could probably be removed from this one. I don't think it will ever make sense to publish it, will it?

Reply:
Done